alcs-api/src/card/card.dto.ts

```diff
@@ -1,12 +1,92 @@
 import { AutoMap } from '@automapper/classes';
+import { IsBoolean, IsNotEmpty, IsOptional, IsString } from 'class-validator';
+import { UserDto } from '../user/user.dto';
+import { CardStatusDto, CardTypeDto } from './card-status/card-status.dto';
 
 export class CardUpdateDto {
   @AutoMap()
+  @IsString()
+  @IsNotEmpty()
+  uuid: string;
```

Comment:
Since you can't change uuid, I prefer it not to be part of the UpdateDto and accepted by the param.